### PR TITLE
setup apollo server to use context object with user info

### PR DESCRIPTION
**Pull request by devnguy**

### CODE REVIEW

File: src/schema/resolvers.js [hunk 1]
```diff
@@ -28,13 +25,14 @@ const resolvers = {
...
       return await prisma.user.create({
         data: {
-          name: args.input.name,
-          email: args.input.email,
-          type: args.input.type,
-          password: args.input.password,
+          name: input.name,
+          email: input.email,
+          type: input.type,
+          password: hash,
```
Comment (RebekahKoon) [inline src/schema/resolvers.js:35]: Probably also need to create the resume here: 

```
data: {
    name: input.name,
    email: input.email,
    type: input.type,
    password: input.password,
    // Creating empty resume
    resume: {
        create: {},
    },
}
```

File: src/schema/resolvers.js [hunk 2]
```diff
@@ -110,21 +108,10 @@ const resolvers = {
...
-      const user = tempData[tempData.length - 1]
-
-      return user
-    },
-
     loginUser: async (_, { email, password }, ctx) => {
-      // const user = await ctx.prisma.user({ email })
-      // Use this while waiting for prisma
-      const [user] = tempData.filter((user) => user.email === email)
+      const user = await prisma.user.findFirst({
```
Comment (RebekahKoon) [inline src/schema/resolvers.js]: It may be better to use findUnique here since email will be unique.

**Review by RebekahKoon — COMMENTED** [review]
The authentication changes look good! I added a few comments about possible changes. I'll leave it up to you if you want to implement them.

**Review by RebekahKoon — DISMISSED** [review]

**Comment by devnguy** [comment]
Was temporarily using findFirst since email wasn't unique at the time. Thanks for pointing it out.

**Review by RebekahKoon — APPROVED** [review]
Everything looks good!

**Pull request merged**